sql/moz-fx-data-shared-prod/firefox_accounts/fxa_all_events/view.sql

```diff
@@ -3,80 +3,178 @@ CREATE OR REPLACE VIEW
 ),
   -- This table doesn't include any user events that are considered "active",
   -- but should always be included for a complete raw event log.
 fxa_auth_bounce_events AS (
   SELECT
-    timestamp AS submission_timestamp,
+    `timestamp`,
+    receiveTimestamp,
     jsonPayload.fields.user_id,
     CAST(
       NULL AS STRING
     ) AS country,  -- No country field in auth_bounces
     jsonPayload.fields.language,
     jsonPayload.fields.app_version,
-    CAST(NULL AS STRING),
-    CAST(NULL AS STRING),
+    CAST(NULL AS STRING) AS os_name,
+    CAST(NULL AS STRING) AS os_version,
     jsonPayload.fields.event_type,
-    JSON_VALUE(jsonPayload.fields.event_properties, '$.service') AS service
+    jsonPayload.logger,
+    jsonPayload.fields.user_properties,
+    jsonPayload.fields.event_properties,
+    CAST(NULL AS STRING) AS device_id,
   FROM
     `moz-fx-data-shared-prod.firefox_accounts_derived.fxa_auth_bounce_events_v1`
 ),
 fxa_content_events AS (
   SELECT
-    timestamp AS submission_timestamp,
+    `timestamp`,
+    receiveTimestamp,
     jsonPayload.fields.user_id,
     jsonPayload.fields.country,
     jsonPayload.fields.language,
     jsonPayload.fields.app_version,
     jsonPayload.fields.os_name,
     jsonPayload.fields.os_version,
     jsonPayload.fields.event_type,
-    JSON_VALUE(jsonPayload.fields.event_properties, '$.service') AS service
+    jsonPayload.logger,
+    jsonPayload.fields.user_properties,
+    jsonPayload.fields.event_properties,
+    jsonPayload.fields.device_id,
   FROM
     `moz-fx-data-shared-prod.firefox_accounts_derived.fxa_content_events_v1`
 ),
+--
 fxa_oauth_events AS (
   SELECT
-    timestamp AS submission_timestamp,
+    `timestamp`,
+    receiveTimestamp,
     jsonPayload.fields.user_id,
     CAST(NULL AS STRING) AS country,
     CAST(NULL AS STRING) AS language,
     jsonPayload.fields.app_version,
     CAST(NULL AS STRING) AS os_name,
     CAST(NULL AS STRING) AS os_version,
     jsonPayload.fields.event_type,
-    JSON_VALUE(jsonPayload.fields.event_properties, '$.service') AS service
+    jsonPayload.logger,
+    jsonPayload.fields.user_properties,
+    jsonPayload.fields.event_properties,
+    CAST(NULL AS STRING) AS device_id,
   FROM
     `moz-fx-data-shared-prod.firefox_accounts_derived.fxa_oauth_events_v1`
+),
+stdout AS (
+  SELECT
+    `timestamp`,
+    receiveTimestamp,
+    jsonPayload.fields.user_id,
+    CAST(NULL AS STRING) AS country,
+    jsonPayload.fields.language,
+    jsonPayload.fields.app_version,
+    jsonPayload.fields.os_name,
+    jsonPayload.fields.os_version,
+    jsonPayload.fields.event_type,
+    jsonPayload.logger,
+    jsonPayload.fields.user_properties,
+    jsonPayload.fields.event_properties,
+    jsonPayload.fields.device_id,
+  FROM
+    `moz-fx-data-shared-prod.firefox_accounts_derived.fxa_stdout_events_v1`
+),
+unioned AS (
+  SELECT
+    *,
+    'fxa_auth_event' AS event_category,
```

Comment:
IMO the `fxa_` prefixes and `_event` suffixes on the event categories are kind of superfluous, and I'd suggest simplifying them to just `auth`, `auth_bounce`, etc.

Reply:
Made the change. 🙇